docs/API-functions.md

```diff
@@ -645,14 +645,25 @@ where:
 * `result`: result of the operation (if this operation recently updated this structure)
 * `srcaddr`: address to which `id` should be bound
 * `peeraddr`: address to which `id` should be connected
-* `priority`: priority for backup group
-
-The priority is set to 0 by default by `srt_prepare_endpoint()` - you can set
-it to a different value afterwards. The default 0 value is the highest priority
-and greater values declare lower priorities. The priority for the backup
-groups determines which link is activated first when the currently active link is
-unstable, and which should keep transmitting when multiple active links are
-currently stable. This is not used by any other group types.
+* `weight`: weight value
+
+The weight is set to 0 by default by `srt_prepare_endpoint()` - you can set
+it to a different value afterwards. The meaning of weight depends on the group
+type:
+
+1. Backup groups: in this case it defines the link priority. The default 0
+value is the highest priority and greater values declare lower priorities. The
+priority for the backup groups determines which link is activated first when
+the currently active link is unstable, and which should keep transmitting when
+multiple active links are currently stable.
+
+2. Balancing groups with "fixed" algorighm: in this case it defines the
+desired link load share. You can think of it as a percentage of link load,
+but indeed a load percentage is defined as this weight value divided by a sum
+of all weight values from all member links. Note however that the sum is
+calculated out of all links that have been successfully connected. The
+default 0 is also a special value that defines an "equalized" load share
+(its set to the arithmetic average of the weights from all links).
```

Comment:
```suggestion
(it is set to the arithmetic average of the weights from all links).
```

Reply:
The original text has changed here again, so I reused your text with:

"weight values. Additionally, a value of 0 is special and it is translated
into the arithmetic average of all non-zero weighted links."